src/python/grpcio/tests/unit/_adapter/_intermediary_low_test.py

```diff
@@ -302,7 +302,8 @@ def testOneManyByteEcho(self):
     self._perform_echo_test([_BYTE_SEQUENCE])
 
   def testManyOneByteEchoes(self):
-    self._perform_echo_test(_BYTE_SEQUENCE)
+    self._perform_echo_test(
```

Comment:
This one took me a little while to figure out.  In Python3, the default iterator over a byte string is an iterator of the integer value of each byte.  This turns the byte string into a byte array, which gives us what we want in python 2 and 3.